### DISCUSSION TITLE
I use Anura3D to analysis rainfall induced landslide ...when i conduct calculation the following error comes, can any one tell me the reason

### DISCUSSION THREAD

**Discussion by girmameaza12** [Anura3D/Anura3D_OpenSource#140] [Q&A] [answered]
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/141491544/0d29f3fa-2292-41b6-a822-a3eaf8c3130a)

**Comment by FrancescaCeccato** [comment, marked as answer]
This error means that you have an error in the GOM file. Maybe some input is not defined correctly. You should rebuild the model.
